.github/workflows/build_steps.yml

```diff
@@ -9,6 +9,7 @@ on:
       cibw_version:      {required: false, type: string, description: Follower only. Must match the cibw_image_tag}
       python_deps_ids:   {default: '[""]', type: string, description: Follower test matrix parameter. JSON string.}
       python3:           {default: -1, type: number, description: Python 3 minor version}
+      real_storage:      {required: false, default: "false", type: string, description: Specifies wheather the python tests should tests against real storages e.g. AWS S3  }
```

Comment:
This is a neat way to do it. Should make it easy to run more often if we discover it isn't too expensive.